test/e2e/specs/editor/various/patterns.spec.js

```diff
@@ -31,18 +32,18 @@ test.describe( 'Unsynced pattern', () => {
 		await createPatternDialog
 			.getByRole( 'textbox', { name: 'Name' } )
 			.fill( 'My unsynced pattern' );
-		await page.getByLabel( 'Synced' ).click();
+		await createPatternDialog.getByRole( 'checkbox' ).setChecked( false );
```

Comment:
It should also have the `Synced` name:

```suggestion
		await createPatternDialog.getByRole(
			'checkbox',
			{ name: 'Synced' }
		).setChecked( false );
```